Poc faust integration (#282)
ohmtech-rdi/eurorack-blocks: documentation/faust/mapping.md

# Name Mapping System

> In the previous chapter, we've seen that we can have a FAUST primitive with a label, and use

Author: Yes, all in all, I think we need to make probably a second pass on the documentation, to make sure that we are using the right FAUST terminology or taxonomy. I will for example change "audio object" to "dsp function" like we discussed this morning, but I guess there are some other cases there and there.

> In the previous chapter, we've seen that we can have a FAUST primitive with a label, and use

Reviewer: OK

      label "OUT L"
   }

   control audio_out2 AudioOut {
      position 8hp, 111mm
      style thonk.pj398sm.knurled
      label "OUT R"
   }
```

Now, let's build the project. If you go to your IDE build logs, you will see the following:

```
warning: non mapped checkbox /PHASER2/0x00/Bypass
warning: non mapped checkbox /PHASER2/0x00/Invert Internal Phaser Sum
warning: non mapped checkbox /PHASER2/0x00/Vibrato Mode
warning: non mapped hslider /PHASER2/0x00/Speed
warning: non mapped hslider /PHASER2/0x00/Notch Depth (Intensity)
warning: non mapped hslider /PHASER2/0x00/Feedback Gain
warning: non mapped hslider /PHASER2/0x00/Notch width
warning: non mapped hslider /PHASER2/0x00/Min Notch1 Freq
warning: non mapped hslider /PHASER2/0x00/Max Notch1 Freq
warning: non mapped hslider /PHASER2/0x00/Notch Freq Ratio: NotchFreq(n+1)/NotchFreq(n)
warning: non mapped hslider /PHASER2/0x00/Phaser Output Level
```

```{note}
If your IDE doesn't show message logs or accessing them is not convenient,
just can also build with `erbb build simulator` instead from your terminal.
```

They represent the list of all values in FAUST that do not have a mapping in `erbui`.

> `/PHASER2/0x00/Bypass` represents an address in FAUST.
> This is what is used to control FAUST using OSC for example.

Author: ```suggestion
`/PHASER2/0x00/Bypass` represents an address in FAUST,
which can be used with OSC for example.
```
